server/bleep/src/webserver.rs

```diff
@@ -79,6 +79,7 @@ pub async fn start(app: Application) -> anyhow::Result<()> {
         .route("/studio", get(studio::list))
         .route("/studio/:id", get(studio::get))
         .route("/studio/:id", patch(studio::patch))
+        .route("/studio/:id", delete(studio::delete))
```

Comment:
could compress these:
```suggestion
        .route("/studio/:id", get(studio::get).patch(studio::patch).delete(studio::delete))
```

Reply:
Good point :slightly_smiling_face: